lib/index.js

```diff
@@ -19,6 +19,25 @@ function spawnSeal(){
   }
 }
 
+function moveInput() {
+  window.addEventListener('keydown', function(e) {
+    if (e.keyCode === 37) {
+      narwhal.x -= 5;
+    } else if (e.keyCode === 38) {
+      narwhal.y -=5;
+    } else if (e.keyCode === 40) {
+      narwhal.y += 5;
+    } else if (e.keyCode === 39) {
+      narwhal.x += 5;
+    }
+      console.log(e.keyCode);
+  })
+}
+
+moveInput();
+
```

Comment:
Looks like it is moving the narwhal. I agree that we need to figure out how to smooth the movement.